src/main/java/org/lsc/Hooks.java

```diff
@@ -63,31 +64,56 @@ public class Hooks {
 	 *
 	 * return nothing
 	 */
-	public final static void postSyncHook(final String hook, final LscModifications lm) {
+	public final static void postSyncHook(final String hook, final String outputFormat, final LscModifications lm) {
 
 		if( hook != null && ! hook.equals("") )
 		{
-			// Compute json modifications
-			String jsonModifications = null;
+
+			String format = "";
+			if( outputFormat.equals("json") ) {
```

Comment:
the parsing of the outputFormat should be done previously, here you should be able to manipulate only an Enum